app/src/main/java/com/sopt/clody/presentation/ui/diarylist/component/DailyDiaryCard.kt

```diff
@@ -20,34 +20,28 @@ import androidx.compose.material3.Card
 import androidx.compose.material3.CardDefaults
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import com.sopt.clody.R
 import com.sopt.clody.data.remote.dto.response.ResponseMonthlyDiaryDto
-import com.sopt.clody.presentation.ui.component.bottomsheet.DiaryDeleteSheet
-import com.sopt.clody.presentation.ui.component.dialog.ClodyDialog
+import com.sopt.clody.presentation.ui.diarylist.screen.DiaryListViewModel
 import com.sopt.clody.ui.theme.ClodyTheme
-import java.time.LocalDate
-import java.time.format.TextStyle
-import java.util.Locale
 
 @Composable
 fun DailyDiaryCard(
     index: Int,
+    diaryListViewModel: DiaryListViewModel,
     dailyDiary: ResponseMonthlyDiaryDto.DailyDiary,
+    year: Int,
+    month: Int,
+    day: Int,
+    dayOfWeek: String,
```

Comment:
[p2] 사실 저도 적용해야 하는 부분인데...ㅎ 매개변수가 너무 많아서 가독성이 떨어지는 문제가 있습니다! 그래서 데이터를 하나의 객체로 묶어 전달하는 방법은 어떨까요? 
```suggestion
   data class DailyDiaryCardData(
    val index: Int,
    val dailyDiary: ResponseMonthlyDiaryDto.DailyDiary,
    val year: Int,
    val month: Int,
    val day: Int,
    val dayOfWeek: String
)
```

Reply:
안그래도 DiaryListViewModel 밑에 부분에 보시면 묶어서 사용하려는 노력을 했으나.. ui 구조상 생각보다 쉽지 않더라구요. 기능 구현 완료되면 제일 우선적으로 리팩 해보겠습니다!